packages/react-router/.eslintrc.js

```diff
@@ -1,8 +1,13 @@
 let restrictedGlobalsError = `Node globals are not allowed in this package.`;
 
 module.exports = {
-  extends: "../../.eslintrc",
+  env: {
+    browser: true,
+    commonjs: true,
+  },
   rules: {
+    strict: 0,
+    "no-restricted-syntax": ["error", "LogicalExpression[operator='??']"],
```

Comment:
This is just porting across equivalent config from `.eslintrc` since it was converted to `.eslintrc.js` we it can include the options from the  `server-runtime` package.